Dockerfile

```diff
@@ -29,13 +29,15 @@ RUN apt-get update && \
     net-tools \
     netcat \
     openjdk-8-jdk-headless \
+    psmisc \ 
     python \
     python-pip \
     python3 \
     python3-pip \
     sudo \
     tzdata \
     unzip \
+    vim \
```

Comment:
Why do we need vim? Is this for your local development and debugging?

Reply:
Yeah, just for testing locally. Removed.